### DISCUSSION TITLE
[Bug]: How to disable infinite scan?

### DISCUSSION THREAD

**Discussion by IntegrityPerson** [winstxnhdw/lc-hax#401] [Q&A] [answered]
### What happened?

How to disable infinite scan, as in scanning range infinite?

### Current Commit Hash

a1b2c3d4

### Injector

- [X] SharpMonoInjectorCore
- [X] Others

### Acknowledgement

- [X] I have confirmed that my anti-virus is not blocking any of the relevant programs
- [X] I have done my due diligence to check for similar issues

**Comment by IntegrityPerson** [comment]
Also picking up 2 handed? and not shotguns while lobby is loading

**Comment by IntegrityPerson** [comment]
*Not picking up shotguns while lobby is in orbit and people can join?

**Comment by winstxnhdw** [comment, marked as answer]
Delete the `InfiniteScanPatch`. The rest can also be found in `Patches`. Why not just.. you know.. not pick up a shotgun in orbit?

**Comment by IntegrityPerson** [comment]
Because usually when you try to pick up a shotgun whilie the lobby is accepting new people, it will not let you. Saying: "Please wait until the ship has landed" or something like that. It's something in the mod but I'm not sure where
